examples/window-app/common/src/WindowApp.cpp

```diff
@@ -513,8 +532,15 @@ void WindowApp::Cover::TiltUp()
 
 void WindowApp::Cover::TiltDown()
 {
-    uint16_t percent100ths = 0;
-    Attributes::CurrentPositionTiltPercent100ths::Get(mEndpoint, &percent100ths);
+    EmberAfStatus status;
+    chip::app::DataModel::Nullable<chip::Percent100ths> current;
+    chip::Percent100ths percent100ths = 5000; // set at middle
+
+    status = Attributes::CurrentPositionTiltPercent100ths::Get(mEndpoint, current);
```

Comment:
There are many repeats of this code...  I wonder whether a helper that takes a function pointer and returns a Percent100ths value is worth it....

Reply:
Yes I know Boris , none of this is specs compliant neither clean in the way is done
This is why i have many PRs to pass to make it specs compliants and more factorized/logical and your inputs are warmly welcomed
The core of WindowCovering  cluster is broken we both know that ... we will need to find a better long solution

The goal of this PR is just to update the XML -> Other PR will remove most of this code